x-pack/solutions/search/plugins/search_homepage/public/components/connect_to_elasticsearch.tsx

```diff
@@ -66,6 +69,16 @@ export const ConnectToElasticsearch = () => {
               </EuiFlexItem>
             </EuiFlexGroup>
           </EuiFlexItem>
+          {isQuickstartEnabled && (
+            <EuiFlexItem grow={false}>
+              <EuiHorizontalRule />
+            </EuiFlexItem>
+          )}
+          {isQuickstartEnabled && (
+            <EuiFlexItem grow={false}>
+              <QuickstartsGroup />
+            </EuiFlexItem>
+          )}
```

Comment:
non-blocking nitpick:
```suggestion
          {isQuickstartEnabled && (
          <>
            <EuiFlexItem grow={false}>
              <EuiHorizontalRule />
            </EuiFlexItem>
            <EuiFlexItem grow={false}>
              <QuickstartsGroup />
            </EuiFlexItem>
          </>
          )}
```